resources/prometheus/prometheus-adapter-clusterRoleAggregatedMetricsReader.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Is this resource coming from an external (prometheus) repository? Just trying to understand what it's like to update it for different kubernetes versions.

Reply:
Technically this resource is not used in this release. Just backported and later I realized I have no bindings to it anyway.